tests/fixed/doc-comment-spacing.php

```diff
@@ -56,10 +56,14 @@ public function c(iterable $foo): void
      * @PHPCR\Uuid
      * @PHPCR\Field
      *
-     * @param int[] $foo
-     * @param int[] $bar
+     * @param         int[] $foo
+     * @param         int[] $bar
```

Comment:
Is it intended that parameter types are aligned regardless of the vendor prefix but the names aren't? Since we assume prefixed and non-prefixed to have the same meaning, I'd expect them to be aligned in the same way:
```php
* @param         int[]              $foo
* @param         int[]              $bar
* @psalm-param   array<string, int> $foo
* @phpstan-param array<string, int> $foo
```

Reply:
Hi @morozov we discussed this here https://github.com/doctrine/coding-standard/pull/238#discussion_r594103519
It is not possible from the phpcs package.

Reply:
If you are interested in having this formatting I am pretty sure adding another configuration option to phpcs shouldn't be too hard. But from the structure of the code I don't want to spend time getting into it to contribute there.

Reply:
Yes it's not the goal of this PR, would be a scope creep